docs/docs/dev_docs/updating.md

```diff
@@ -66,32 +66,29 @@ aztec-cli update . --contract src/contract1 --contract src/contract2
 
 ### Manual update
 
-Finally we need to update the Noir framework for Aztec contracts.
-We need to install a version compatible with our `nargo` and Sandbox.
-
-To update the framework we will update a tag of the `aztec.nr` dependency in the `Nargo.toml` file to the `SANDBOX_VERSION` from above.
-Find all the dependencies pointing to the directory within `aztec.nr` framework and update the corresponding tag.
-E.g.:
+To update the aztec.nr packages manually, update the tags of the `aztec.nr` dependencies in the `Nargo.toml` file.
 
 ```diff
 [dependencies]
 -aztec = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v0.7.5", directory="yarn-project/aztec-nr/aztec" }
-+aztec = { git="https://github.com/AztecProtocol/aztec-packages", tag="#include_aztec_version", directory="yarn-project/aztec-nr/aztec" }
++aztec = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v0.16.9", directory="yarn-project/aztec-nr/aztec" }
 -value_note = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v0.7.5", directory="yarn-project/aztec-nr/value-note" }
-+value_note = { git="https://github.com/AztecProtocol/aztec-packages", tag="#include_aztec_version", directory="yarn-project/aztec-nr/value-note" }
++value_note = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v0.16.9", directory="yarn-project/aztec-nr/value-note" }
 ```
 
-Go to the project directory and try compiling it with `aztec-cli` to verify that the update was successful:
+Go to the contract directory and try compiling it with `aztec-nargo compile` to verify that the update was successful:
 
 ```shell
-cd /your/project/root
-aztec-cli compile ./
+cd /your/contract/directory
+aztec-nargo compile
 ```
 
 If the dependencies fail to resolve ensure that the tag matches a tag in the [aztec-packages repository](https://github.com/AztecProtocol/aztec-packages/tags).
 
-## Updating `nargo`
+## Updating `aztec-nargo`
 
-Nargo is not strictly required, but you may want to use it for the LSP or testing. More info [here](./getting_started/aztecnr-getting-started.md#install-nargo-recommended).
+`aztec-nargo` is updated by running:
 
-<InstallNargoInstructions />
+```bash
+aztec-up latest
```

Comment:
does it need "latest"? TIL - I have been using it without tbh

Reply:
oh i get an error if not
![image](https://github.com/AztecProtocol/aztec-packages/assets/18372439/82302163-6436-45b5-a403-69a81b9a7809)